packages/ibm-products/src/components/Datagrid/Datagrid/DatagridRow.js

```diff
@@ -183,16 +184,20 @@ const DatagridRow = (datagridState) => {
           );
           return (
             <TableCell
-              className={cx(`${blockClass}__cell`, {
-                [`${blockClass}__expandable-row-cell`]:
-                  row.canExpand && index === 0,
-                [`${blockClass}__expandable-row-cell--is-expanded`]:
-                  row.isExpanded && index === 0,
-                [`${blockClass}__slug--cell`]:
-                  associatedHeader &&
-                  associatedHeader.length &&
-                  isValidElement(associatedHeader[0]?.slug),
-              })}
+              className={cx(
+                `${blockClass}__cell`,
+                {
+                  [`${blockClass}__expandable-row-cell`]:
+                    row.canExpand && index === 0,
+                  [`${blockClass}__expandable-row-cell--is-expanded`]:
+                    row.isExpanded && index === 0,
+                  [`${blockClass}__slug--cell`]:
+                    associatedHeader &&
+                    associatedHeader.length &&
+                    isValidElement(associatedHeader[0]?.slug),
+                },
+                `${columnClassname}`
```

Comment:
the template strings here aren't necessary `columnClassname`

Reply:
Updated